tests/unit/test_httpchecksum.py

```diff
@@ -536,6 +546,13 @@ def test_handle_checksum_body_checksum_skip_streaming(self):
         body = response_dict["body"]
         self.assertEqual(body.read(), b"hello world")
 
+    @requires_crt()
+    def test_checksum_cls_crt_supported_algorithms_in_sync(self):
+        self.assertEqual(
+            sorted(_CRT_SUPPORTED_CHECKSUM_ALGORITHMS),
+            sorted(list(_CRT_CHECKSUM_CLS.keys())),
+        )
+
```

Comment:
Just curious, can you explain the preference of calling the assertion at runtime instead of exposing a private variable in the test suite? Is this just a design principle or is there potential risk to exposing private attributes outside of runtime?